deploy.json

```diff
@@ -43,14 +43,28 @@
       "add-header-comment": true
     },
 
-    "// server - API tests",
+    "// server",
     {
       "type": "move",
       "src": "src/server/",
       "dst": "[[package]]/server/",
       "match": "^.*\\.(py)$",
       "add-header-comment": true
     },
+    {
+      "type": "move",
+      "src": "src/server/endpoints",
+      "dst": "[[package]]/server/endpoints/",
+      "match": "^.*\\.(py)$",
+      "add-header-comment": true
+    },
+    {
+      "type": "move",
+      "src": "src/server/endpoints/covid_utils",
```

Comment:
isn't it called covidicast_utils? https://github.com/cmu-delphi/delphi-epidata/tree/sgratzl/flaskx/src/server/endpoints/covidcast_utils

Reply:
```suggestion
      "src": "src/server/endpoints/covidcast_utils",
```